docs/backend/upgrading/version-specific-migration/upgrade-to-61.md

```diff
@@ -0,0 +1,58 @@
+---
+myst:
+  html_meta:
+    "description": "How to upgrade to Plone 6.1"
+    "property=og:description": "How to upgrade to Plone 6.1"
+    "property=og:title": "How to upgrade to Plone 6.1"
+    "keywords": "Upgrade, Plone 6"
+---
+
+(backend-upgrade-plone-v61-label)=
+
+# Upgrade Plone 6.0 to 6.1
+
+Plone 6.1 has seen the following major changes.
+Some may require changes in your setup.
+
+
+## TinyMCE upgraded
+
+Plone 6.0 uses TinyMCE, a rich text editor for websites.
+TinyMCE 5 reached its end of support on April 20, 2023.
+For Plone 6.1, TinyMCE has been upgraded from version 5 to 7.
+
+```{seealso}
+-   [How to upgrade TinyMCE 5 to TinyMCE 6](https://www.tiny.cloud/blog/upgrade-to-tinymce-6/)
+-   [Upgrading TinyMCE](https://www.tiny.cloud/docs/tinymce/latest/upgrading/)
```

Comment:
This is a bit unclear: If I maintain a Plone site, do I need to take the time to read and understand the info in these links, or were the necessary steps already taken by the Plone classic UI maintainers?

Also this section should mention that the change is only relevant for sites using the classic UI.

Reply:
Yes this needs more details. Something like:

There's no action needed if you upgrade a standard Classic-UI 6.0 Site to 6.1 (=TinyMCE 7) unless you have custom plugins implemented or you use a plugin which got removed (or moved to premium) in Tiny 6/7. To upgrade your plugin implementation to Tiny 7 please check the upgrade guides here https://www.tiny.cloud/docs/tinymce/6/migration-from-5x/#plugins

Reply:
Revised in https://github.com/plone/documentation/pull/1655/commits/2b66e7ad90476ae38058db08d4f002798e45a67b. Please take another look.